app/Repositories/Eloquents/PostEloquentRepository.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/4/18
+ * Time: 6:21 PM
+ */
+
+namespace App\Repositories\Eloquents;
+
+
+use App\Models\Post;
+use App\Repositories\Contracts\PostRepository;
+
+class PostEloquentRepository extends AbstractEloquentRepository implements PostRepository
+{
+
+    public function model()
+    {
+        // TODO: Implement model() method.
+        return new Post;
+    }
+
+    public function getAll($data = [])
+    {
+        // TODO: Implement getAll() method.
+        return $this->model()->all();
+    }
+
+    public function show($id)
+    {
+        // TODO: Implement show() method.
+        $data = $this->model()->findOrFail($id);
+
+        return $data;
+    }
+
+    public function delete($id)
+    {
+        // TODO: Implement delete() method.
+        $model = $this->model()->findOrFail($id);
+
+        return $model->delete();
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostEloquentRepository.php</br>**Line**: 45</br>Expected 1 newline at end of file; 0 found